sdk-python/littlehorse/workflow.py

```diff
@@ -634,6 +639,26 @@ def execute(self, task_name: str, *args: Any) -> NodeOutput:
         node_name = self.add_node(task_name, task_node)
         return NodeOutput(node_name)
 
+    def assign_user_task(
```

Comment:
Some methods are missing right? and an example

Reply:
Yes and yes. Still in draft...